.pylintrc

```diff
@@ -150,7 +150,8 @@ disable=print-statement,
         comprehension-escape,
         line-too-long,
         fixme,
-        fixme-info
+        fixme-info,
+	duplicate-code
```

Comment:
It seems that we have some indent inconsistency here.

Reply:
ah, default to tabs is what I have :-)
Fixed